dipy/viz/window.py

```diff
@@ -486,6 +469,52 @@ def start(self):
         del self.iren
         del self.window
 
+    def record(self, filename="record.log"):
```

Comment:
After thinking about it, I will simply rename the methods `record` to `record_events` and `play` to `play_events`. However, I think that adding a parameter `mode` to the `start` method won't be that useful and it would also require adding a `filename` parameter (which is mandatory for both `record_events` and `play_events`).